subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -886,135 +921,24 @@ What happens when we declare that "google-collections" are replaced by "guava"?
-
-[[sec:cache_command_line_options]]
-==== Command line options to override caching
-
-
-[[sub:cache_offline]]
-===== Offline
-
-The `--offline` command line switch tells Gradle to always use dependency modules from the cache, regardless if they are due to be checked again. When running with offline, Gradle will never attempt to access the network to perform dependency resolution. If required modules are not present in the dependency cache, build execution will fail.
-
-[[sub:cache_refresh]]
-===== Refresh
-
-At times, the Gradle Dependency Cache can be out of sync with the actual state of the configured repositories. Perhaps a repository was initially misconfigured, or perhaps a “non-changing” module was published incorrectly. To refresh all dependencies in the dependency cache, use the `--refresh-dependencies` option on the command line.
-
-The `--refresh-dependencies` option tells Gradle to ignore all cached entries for resolved modules and artifacts. A fresh resolve will be performed against all configured repositories, with dynamic versions recalculated, modules refreshed, and artifacts downloaded. However, where possible Gradle will check if the previously downloaded artifacts are valid before downloading again. This is done by comparing published SHA1 values in the repository with the SHA1 values for existing downloaded artifacts.
-
-[[sec:controlling_caching]]
-==== Fine-tuned control over dependency caching
-
-You can fine-tune certain aspects of caching using the `ResolutionStrategy` for a configuration.
-
-By default, Gradle caches dynamic versions for 24 hours. To change how long Gradle will cache the resolved version for a dynamic version, use:
-
-++++
-<sample id="dynamic-version-cache-control" dir="userguide/artifacts/resolutionStrategy" title="Dynamic version cache control">
-                <sourcefile file="build.gradle" snippet="dynamic-version-cache-control"/>
-            </sample>
-++++
-
-By default, Gradle caches changing modules for 24 hours. To change how long Gradle will cache the meta-data and artifacts for a changing module, use:
-
-++++
-<sample id="changing-module-cache-control" dir="userguide/artifacts/resolutionStrategy" title="Changing module cache control">
-                <sourcefile file="build.gradle" snippet="changing-module-cache-control"/>
-            </sample>
-++++
-
-For more details, take a look at the API documentation for api:org.gradle.api.artifacts.ResolutionStrategy[].
-
-[[sec:dependency_management_overview]]
-=== Dependency management best practices
+[[sec:troubleshooting_dependency_resolution]]
+=== Troubleshooting dependency resolution
 
 While Gradle has strong opinions on dependency management, the tool gives you a choice between two options: follow recommended best practices or support any kind of pattern you can think of. This section outlines the Gradle project's recommended best practices for managing dependencies.
```

Comment:
I recognize this isn't part of your change, but I'm not sure of the value of these 2 paragraphs. I think one introductory sentence would be sufficient.

Reply:
done